packages/salesforcedx-test-utils-vscode/src/testrunner.ts

```diff
@@ -131,7 +133,12 @@ function run(testsRoot: any, clb: any): any {
           .on(
             'fail',
             (test: any, err: any): void => {
-              console.log(`Failure in test '${test}': ${err}`);
+              if (typeof test === 'object') {
```

Comment:
If it fails for a reason beyond a test, such as the EACCES error, it'll show "object object" not the issue.